webpack.config.js

```diff
@@ -42,7 +45,17 @@ module.exports = function (options, webpackOptions) {
         { context: './node_modules/monaco-editor/min/', from: '**/*', to: 'monaco' }
       ]),
     ],
-    stats: 'minimal'
+    stats: { 
```

Comment:
`stats` could be defined earlier as a var and then used here and at line #91.

Reply:
done.